.prettierignore

```diff
@@ -17,6 +17,10 @@ Procfile
 /deployment/node_modules/
 # white-list files we want to process
 !*.js
+!*.jsx
+!*.html
+!*.css
```

Comment:
Remove this. If we have any, they're not ours but third-party stuff we've checked in.